tests/unit/src/main/scala/tests/BaseWorksheetLspSuite.scala

```diff
@@ -469,9 +471,11 @@ abstract class BaseWorksheetLspSuite(
           |a.Util.increase(1) // : Int = 2
           |""".stripMargin,
       )
-      _ <- server.didSave("a/src/main/scala/a/Util.scala")(
+      _ <- server.didChange("a/src/main/scala/a/Util.scala")(
         _.replace("n + 1", "n + 2")
       )
+      _ <- server.didSave("a/src/main/scala/a/Util.scala")(identity)
+      _ <- server.didFocus("a/src/main/scala/a/Main.worksheet.sc")
```

Comment:
I don't think we should ever need `didFocus`, since it may not be supported, right?

Reply:
It's just focus switched to Util.scala and the worksheet doesn't evaluate. This will work in VS Code, thoguh wonder what happens if did focus is not supported